### DISCUSSION TITLE
please make the old forum searchable

### DISCUSSION THREAD

**Discussion by 3muchfun** [GetPublii/Publii#1507] [General] [answered]
Hi, why is the old forum not searchable? There surely are some tips&tricks written down by users and even the programmers like @bobmitro  or @dziudek which would answer questions without asking them again on GitHub...

**Comment by gpsblues** [comment, marked as answer]
Hi,

with Google you can try this query:
**site:forum.getpublii.com  _keyword1 keyword2 keyword3 ..._**

**Reply by 3muchfun** [reply]
Thanks, @gpsblues, for the idea. It does not answer y question, but I will give it a try.
